src/pages/postgraphile/required-knowledge.md

```diff
@@ -176,3 +176,44 @@ create policy friend_insert on my_other_table for insert with check ( are_friend
   that action (doesn't make sense for `SELECT` or `DELETE`)
 - NOTE: if `with check` is not specified then the `using` clause will also be
   used for `with check` automatically.
+
+#### Recommended path
+
+A good way to tackle and make a sense of PostGraphile is to go through these
+steps:
+
+**Introduction**
+
+https://www.graphile.org/postgraphile/introduction/
+
+Describes why you would want to go with this approach in the first place and
+what the benefits and features of using PostGraphile are. Do not be deterred if
+you find this daunting and are not used to using this much `SQL`. Using the
+database as the application layer is a fairly modern and unfamiliar concept,
+but by adopting it there are many benefits you may reap by being close to the
+data, as well as leveraging the declarative nature of SQL.
+
+**Quick start**
+
+https://www.graphile.org/postgraphile/quick-start-guide/
+
+Walks you through installing the required software such as `node` and `postgres`
+and creating of a database that PostGraphile will be connecting to.
+
+**Schema Design**
+
+https://www.graphile.org/postgraphile/postgresql-schema-design/
+
+Whether you have or have not built databases before, this section gives valuable
+advice about how to design a solid database. Topics include naming conventions, 
+namespacing with schemas, managing access with roles and policies, handling
+migrations and much more. If these words are unfamiliar to you, do not worry, it
+will be clear once you have built a database following this guide.
+
+**Postgraphile-in-docker**
+
+https://www.graphile.org/postgraphile/running-postgraphile-in-docker/
+
+This will run through the making of GraphQL endpoints to build a forum; like
+hackernews. This goes through installation all the way to querying your
+query/mutations built from postgraphile.
```

Comment:
```suggestion
PostGraphile does not require Docker, and is very at home in a pure Node.js
environment, so if you're not big on Docker you can skip this one!
 
If you develop using Docker, this will run through the building of a GraphQL
API for a forum-like website. It goes through installation through to
example queries and mutations to issue to PostGraphile.
```